modules/apps/29-fee/ibc_module_test.go

```diff
@@ -341,108 +333,109 @@ func (suite *FeeTestSuite) TestOnChanCloseInit() {
 			suite.SetupTest()
 			suite.coordinator.Setup(suite.path) // setup channel
 
-			origBal := suite.chainA.GetSimApp().BankKeeper.GetAllBalances(suite.chainA.GetContext(), suite.chainA.SenderAccount.GetAddress())
+			packetID := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 1)
+			fee = types.Fee{
+				RecvFee:    validCoins,
+				AckFee:     validCoins2,
+				TimeoutFee: validCoins3,
+			}
```

Comment:
should we rename `validCoins` to `defaultRecvFee`..etc, to be consistent with keeper tests?

Reply:
I thought about this as well. I think we should, but would prefer to have this be done in a followup since this pr already has a bit of diffs

Reply:
Yeah, no worries. Sounds good to me :)

Reply:
I decided to do it in this pr